app/src/components/Navbar/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface ILinkContainer {
+  name: string;
+  route?: string;
+  isExit?: boolean | undefined;
+  isNoteLink?: boolean | undefined;
```

Comment:
will do the same
if you use it as optional with ? mark, you shoudn't add undefined
```suggestion
  isExit?: boolean;
  isNoteLink?: boolean;
```